tests/configuration_data.py

```diff
@@ -0,0 +1,135 @@
+"""
+    A set of constants describing AP profiles
+"""
+
+APNOS_CREDENTIAL_DATA = {
```

Comment:
To be cleaned up the lab info structure. All of this info must be passed in a single file.

Reply:
So this will be my to do for creating the proper lab info file, but as of now everything works with current stuff. Not much things will be changed with new lab info file structure